kitsune/journal/admin.py

```diff
@@ -0,0 +1,17 @@
+from django.contrib import admin
+
+from kitsune.journal.models import Record
+
+
+class RecordAdmin(admin.ModelAdmin):
```

Comment:
Yes, the admin is the only way to look at these. I think that is good enough for now.

Do you think the model captures enough information? The Fjord one has a lot more details, but I didn't think we needed them.